README.md

```diff
@@ -18,6 +18,8 @@ There are a few things you can configure, this is the most used list:<br/>
 ```OE_VERSION``` is the Odoo version to install, for example ```12.0``` for Odoo V12.<br/>
 ```IS_ENTERPRISE``` will install the Enterprise version on top of ```12.0``` if you set it to ```True```, set it to ```False``` if you want the community version of Odoo 12.<br/>
 ```OE_SUPERADMIN``` is the master password for this Odoo installation.<br/>
+```INSTALL_NGINX``` is set to ```False```  Set this to ```True``` if you want to install Nginx.<br/>
```

Comment:
@tonybenoy has two spaces. Remove one and add a dot. So "```INSTALL_NGINX``` is set to ```False```  Set this to ```True``` if you want to install Nginx.<br/>" should become:
```INSTALL_NGINX``` is set to ```False``` by default. Set this to ```True``` if you want to install Nginx.<br/>